### PR TITLE
feat: implement subscription page using paragon components

### PR DESCRIPTION
**Pull request by long74100**
# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

Screen Shot 2021-10-28 at 10 26 40 AM

Screen Shot 2021-10-28 at 10 34 37 AM

Screen Shot 2021-10-28 at 10 25 12 AM

# Only if submitting a visual change

- [x] Ensure to attach screenshots
- [x] Ensure to have UX team confirm screenshots

### CODE REVIEW

**Review by macdiesel — APPROVED** [review]

File: src/components/subscriptions/data/utils.js [hunk 1]
```diff
@@ -15,3 +19,34 @@ export const getSubscriptionExpiringCookieName = ({
...
+    return SCHEDULED;
+  } if (now.isAfter(subscription.expirationDate)) {
+    return ENDED;
+  }
+
+  return ACTIVE;
+};
+
+// Sort plans by statuses, active -> scheduled -> ended.
+export const sortSubscriptionsByStatus = (subscriptions) => subscriptions.slice().sort(
```
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/components/subscriptions/data/utils.js:36]: @long74100 Curious if `[...subscriptions]` would help clarify that a bit more than using `slice`. Either way is fine with me, though :)

File: src/components/subscriptions/styles/_SubscriptionManagementPage.scss [hunk 2]
```diff
@@ -9,13 +9,45 @@
...
 
+  .subscription-card {
+    margin-bottom: 1.875rem;
+
+    .card-body {
+      background: $light-200;
+    }
+
+    .badge {
+      padding: 0.3rem 0.5rem;
```
Comment (adamstankiewicz) [inline src/components/subscriptions/styles/_SubscriptionManagementPage.scss:26]: Curious why we need to override the default styles on `.badge`? I think we should just use whatever badge styles Paragon gives us by default.

File: src/components/subscriptions/styles/_SubscriptionManagementPage.scss [hunk 3]
```diff
@@ -9,13 +9,45 @@
...
     .card-title {
       @extend .h4;
     }
   }
 
+  .subscription-card {
+    margin-bottom: 1.875rem;
+
+    .card-body {
+      background: $light-200;
```
Comment (adamstankiewicz) [inline src/components/subscriptions/styles/_SubscriptionManagementPage.scss:22]: nit: Could also pass `bg-light-200` as a class name to `Card.Body`.

**Review by adamstankiewicz — COMMENTED** [review]
LGTM. Just a couple non-blocking nits related to the styles.

File: src/components/subscriptions/styles/_SubscriptionManagementPage.scss (hunk 2 above)
Comment (long74100) [reply src/components/subscriptions/styles/_SubscriptionManagementPage.scss:26]: The default badge styles doesn't come with any padding for the badges and it looks quite squished.

Screen Shot 2021-11-02 at 11 23 37 AM
busercontent.com/17792243/139877096-7f962720-7986-40d3-9c1a-77906af19e56.png)
.

Comment (long74100) [reply src/components/subscriptions/styles/_SubscriptionManagementPage.scss]: fixed, issue was with the style import order.

**Pull request merged**